website/docs/my-zombie-flags-report.mdx

```diff
@@ -0,0 +1,47 @@
+---
+id: my-zombie-flags-report
+title: My Zombie Flags Report
+description: This page explains how to use The Zombie Flags Report to list all feature flags that weren't flipped for a while.
+---
+
+Zombie flags are feature flags that are no longer needed or rarely used. Over time, they contribute to technical debt and, if unmanaged, may cause unintended errors or issues.
+
+Here are a few ways zombie flags can form:
+
+- A flag was initially tied to a feature that has since become permanent and is no longer controlled by the flag.
+
+- A decision was made to keep a feature on or off, yet the flag still lingers in your code or DevOps workflow.
+
+- The flag was used for a one-time deployment, but the rollout is over.
+
+- You haven't toggled the flag in a long time.
+
+## Managing Zombie Flags
+
+The Zombie Flags (Stale Flags) Report lists all feature flags that weren't flipped for a while. There is a high chance these flags are not needed anymore but are still referenced in your source code, and ConfigCat recommends removing these flags to avoid tech debt.
+
+### Setting Up the Zombie Feature Flags Report
+
+Login to your [ConfigCat dashboard](https://app.configcat.com), click your Organization name at the top right, then select **My Zombie Flags Report**.
+
+<img alt="Navigating to My Zombie Flags Report" src="/docs/assets/guide/my-zombie-flags-report/navigating-to-my-zombie-flags-report_192dpi.png" width="1224" decoding="async" loading="lazy" />
+
+Here is what the **My Zombie Flags Report** page looks like:
+
+<img alt="My Zombie Flags Report" src="/docs/assets/guide/my-zombie-flags-report/my-zombie-flags-report_192dpi.png" width="1366" decoding="async" loading="lazy" />
+
+Here, you could enable the **Receive report via email** option and select how often the report should be sent by setting the **Frequency**. The **Criteria** option sets how long a flag should be inactive before it is considered stale. Use the **Scope** option to select which flags should be in the report. You can decide if you want the report to include all your feature flags or only the ones you are explicitly watching. Lastly, select the **environment**.
```

Comment:
I think the "Lastly, select the **environment**." part is missing what you will happen when you select the environments.